doc/hyperfine.1

```diff
@@ -0,0 +1,212 @@
+.IP
+This argument requires \fB\-\-parameter\-scan\fR to be specified as well. Traverse the
+range \fImin..max\fP in steps of \fIdelta\fP.
+.IP
+.RS
+Example:
+.RS
+\fBhyperfine\fR \fB\-P\fR delay 0.3 0.7 \fB\-D\fR 0.2 'sleep {delay}'
+.RE
+.RE
+.IP
+This performs benchmarks for 'sleep 0.3', 'sleep 0.5' and 'sleep 0.7'.
+.HP
+\fB\-L\fR, \fB\-\-parameter\-list\fR \fIvar\fP \fIvalues\fP
+.IP
+Perform benchmark runs for each value in the comma\-seperated list of \fIvalues\fP.
+Replaces the string '{\fIvar\fP}' in each command by the current parameter value.
+.IP
+.RS
+Example:
+.RS
+\fBhyperfine\fR \fB\-L\fR compiler gcc,clang '{compiler} \-O2 main.cpp'
+.RE
+.RE
+.IP
+This performs benchmarks for 'gcc \-O2 main.cpp' and 'clang \-O2 main.cpp'.
+.HP
+\fB\-s\fR, \fB\-\-style\fR \fItype\fP
+.IP
+Set output style \fItype\fP (default: auto). Set this to 'basic' to disable output
+coloring and interactive elements. Set it to 'full' to enable all effects even
+if no interactive terminal was detected. Set this to 'nocolor' to keep the
+interactive output without any colors. Set this to 'color' to keep the colors
+without any interactive output. Set this to 'none' to disable all the output
+of the tool.
+.HP
+\fB\-S\fR, \fB\-\-shell\fR \fIshell\fP
+.IP
+Set the \fIshell\fP to use for executing benchmarked commands.
+.HP
+\fB\-i\fR, \fB\-\-ignore\-failure\fR
+.IP
+Ignore non\-zero exit codes of the benchmarked programs.
+.HP
+\fB\-u\fR, \fB\-\-time\-unit\fR \fIunit\fP
+.IP
+Set the time \fIunit\fP to be used. Default: second. Possible values: millisecond, second.
+.HP
+\fB\-\-export\-asciidoc\fR \fIfile\fP 
+.IP
+Export the timing summary statistics as an Asciidoc table to the given \fIfile\fP.
+.HP
+\fB\-\-export\-csv\fR \fIfile\fP
+.IP
+Export the timing summary statistics as CSV to the given \fIfile\fP. If you need the
+timing results for each individual run, use the JSON export format.
+.HP
+\fB\-\-export\-json\fR \fIfile\fP
+.IP
+Export the timing summary statistics and timings of individual runs as JSON to
+the given \fIfile\fP.
+.HP
+\fB\-\-export\-markdown\fR \fIfile\fP
+.IP
+Export the timing summary statistics as a Markdown table to the given \fIfile\fP.
+.HP
+\fB\-\-show\-output\fR
+.IP
+Print the stdout and stderr of the benchmark instead of suppressing it. This
+will increase the time it takes for benchmarks to run, so it should only be
+used for debugging purposes or when trying to benchmark output speed.
+.HP
+\fB\-h\fR, \fB\-\-help\fR
+.IP
+Print help message.
+.HP
+\fB\-V\fR, \fB\-\-version\fR
+.IP
+Show version information.
+.SH EXAMPLES
+.LP
+Basic benchmark of 'find . -iname '*[0-9].jpg'':
+.RS
+.nf
+\fBhyperfine\fR 'find . -iname '*[0-9].jpg''
+.fi
+.RE
+.LP
+Perform benchmarks for 'sleep 0.2' and 'sleep 3.2' with a minimum 5 runs each:
+.RS
+.nf
+\fBhyperfine\fR \fB\-\-min\-runs\fR 5 'sleep 0.2' 'sleep 3.2'
+.fi
+.RE
+.LP
+Perform a benchmark of 'grep' with a warm disk cache by first running the command
+3 times: 
```

Comment:
maybe: "… with a warm disk cache by performing 3 runs upfront that are not part of the measurement"?